### ISSUE TITLE
Does not run from index.html

### ISSUE THREAD

**Issue by brunkowd** [wwt-inc/fish-tank#8]

**Comment by brunkowd** [comment]
After npm install, gulp or gulp serve doesn't appear to work either.

**Comment by mikemcbride** [comment]
I just cloned this repo, `npm install`, then `gulp` and I am not able to reproduce. It's also working on my fork. Is there any chance you didn't git pull after the last PR was merged? @wattsj @hilarie are either of you able to test this?

**Comment by Hilarie** [comment]
It worked for me.

**Comment by dbrunkow** [comment]
@mmcbride1007 I did a clone and no luck.

**Comment by dbrunkow** [comment]
@mmcbride1007 Clone, npm install and no luck. Run node gulp? Load the html page? Nothing happens
What version of node are you running?

**Comment by mikemcbride** [comment]
@dbrunkow 

![node_versions](https://cloud.githubusercontent.com/assets/6248612/9308572/7c2ab242-44ca-11e5-9eee-ee3b904b7694.png)

**Comment by dbrunkow** [comment]
@mmcbride1007 
C02KV26WDR53:fishtank6 brunkowd$ node -v
v0.10.3
C02KV26WDR53:fishtank6 brunkowd$ npm -v
1.4.26
C02KV26WDR53:fishtank6 brunkowd$ gulp -v
-bash: gulp: command not found

**Comment by cvharris** [comment]
@brunkowd try running `npm install -g gulp`

**Comment by mikemcbride** [comment]
Yeah that could be the issue, gulp needs to be installed globally for you
to be able to run it. Same as grunt. We probably should throw that in the
README.

**Comment by dbrunkow** [comment]
Can't get it to work either when building with Jenkins using node .12
Can we ungulp this?
